packages/integration-tests/src/entities/Author.ts

```diff
@@ -122,15 +122,15 @@ config.addRule("books", async (a) => {
 });
 
 // Example of cannotBeUpdated
-config.addRule(cannotBeUpdated("age", (a) => !!a.age && a.age < 100));
+config.addRule(cannotBeUpdated("age"));
```

Comment:
thought: I was half expecting this to look something like 
```suggestion
config.cannotBeChanged("age");
```
But after further digging I see how you are going about it...

And you plan to do use this marker in the future for possible short circuiting 🤔  Interesting.